assignments/mgmt-groups/mg-dts002/assign.appgateway_diagnostics.json

```diff
@@ -0,0 +1,30 @@
+{
+    "properties": {
+        "metadata": {
+            "assignedBy": "github.com/hmcts/azure-policy"
+        },
+        "description": "Forwards all App Gateway Logs to an Azure Event Hub",
+        "displayName": "HMCTS Collect App Gateway Logs",
+        "enforcementMode": "Default",
+        "nonComplianceMessages": [],
+        "notScopes": [],
+        "parameters": {
+            "eventHubAuthRule": {
+                "value": "/subscriptions/8ae5b3b6-0b12-4888-b894-4cec33c92292/resourceGroups/soc-core-infra-prod-rg/providers/Microsoft.EventHub/namespaces/soc-prod-eventhubns/authorizationRules/RootManageSharedAccessKey"
+            }
+        },
+        "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/dts002/providers/Microsoft.Authorization/policyDefinitions/HMCTSAppGatewayDiagnostic",
+        "scope": "/providers/Microsoft.Management/managementGroups/dts002"
+    },
+    "location": "uksouth",
+    "identity": {
+        "type": "UserAssigned"
```

Comment:
Need to specify the resource Id for the UAMI as well, same goes for here https://github.com/hmcts/azure-policy/pull/47/files#diff-d82e491036182a1fc52e240f5339ce165dff86cd5234b41f3ad6bf806426b7b6R21